src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -422,13 +655,19 @@ public final class Data implements Serializable {
     if (o == this) {
       return true;
     }
-    if (o instanceof Data) {
-      Data that = (Data) o;
+    if (o instanceof BillingAccountsGetHttpRequest) {
+      BillingAccountsGetHttpRequest that = (BillingAccountsGetHttpRequest) o;
       return
-
-      this.key.equals(that.getKey())&&
-      this.value.equals(that.getValue())
-      ;
+          this.name.equals(that.getName())&&
+          this.access_token.equals(that.getAccessToken())&&
+          this.callback.equals(that.getCallback())&&
+          this.fields.equals(that.getFields())&&
+          this.key.equals(that.getKey())&&
+          this.prettyPrint.equals(that.getPrettyPrint())&&
+          this.quotaUser.equals(that.getQuotaUser())&&
+          this.userIp.equals(that.getUserIp())&&
+          this.fooRepeated.equals(that.getFooRepeated())
```

Comment:
I think we would want to allow for fields to be null, so this would throw null pointer exceptions. Let's use `Objects.equals()` here.